docs/cog_guides/economy.rst

```diff
@@ -0,0 +1,535 @@
+
+**Description**
+
+Show the user's account balance.
+
+Example:
+    - ``[p]bank balance``
+    - ``[p]bank balance @Twentysix``
+
+**Arguments**
+
+- ``<user>`` The user to check the balance of. If omitted, defaults to your own balance.
+
+.. _economy-command-bank-prune:
+
+""""""""""
+bank prune
+""""""""""
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]bank prune 
+
+**Description**
+
+Base command for pruning bank accounts.
+
+.. _economy-command-bank-prune-global:
+
+"""""""""""""""""
+bank prune global
+"""""""""""""""""
+
+.. note:: |owner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]bank prune global [confirmation=False]
+
+**Description**
+
+Prune bank accounts for users who no longer share a server with the bot.
+
+Cannot be used without a global bank. See ``[p]bank prune server``.
+
+Examples:
+    - ``[p]bank prune global`` - Did not confirm. Shows the help message.
+    - ``[p]bank prune global yes``
+
+**Arguments**
+
+- ``<confirmation>`` This will default to false unless specified.
+
+.. _economy-command-bank-prune-server:
+
+"""""""""""""""""
+bank prune server
+"""""""""""""""""
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]bank prune server [confirmation=False]
+
+.. tip:: Aliases: ``bank prune guild``, ``bank prune local``
+
+**Description**
+
+Prune bank accounts for users no longer in the server.
+
+Cannot be used with a global bank. See ``[p]bank prune global``.
+
+Examples:
+    - ``[p]bank prune server`` - Did not confirm. Shows the help message.
+    - ``[p]bank prune server yes``
+
+**Arguments**
+
+- ``<confirmation>`` This will default to false unless specified.
+
+.. _economy-command-bank-prune-user:
+
+"""""""""""""""
+bank prune user
+"""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]bank prune user <user> [confirmation=False]
+
+**Description**
+
+Delete the bank account of a specified user.
+
+Examples:
+    - ``[p]bank prune user @TwentySix`` - Did not confirm. Shows the help message.
+    - ``[p]bank prune user @TwentySix yes``
+
+**Arguments**
+
+- ``<user>`` The user to delete the bank of. Takes mentions, names, and user ids.
+- ``<confirmation>`` This will default to false unless specified.
+
+.. _economy-command-bank-reset:
+
+""""""""""
+bank reset
+""""""""""
+
+.. note:: |guildowner-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]bank reset [confirmation=False]
+
+**Description**
+
+Delete all bank accounts.
+
+Examples:
+    - ``[p]bank reset`` - Did not confirm. Shows the help message.
+    - ``[p]bank reset yes``
+
+**Arguments**
+
+- ``<confirmation>`` This will default to false unless specified.
+
+.. _economy-command-bank-set:
+
+""""""""
+bank set
+""""""""
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]bank set <to> <creds>
+
+**Description**
+
+Set the balance of user's bank account.
+
+Passing positive and negative values will add/remove currency instead.
```

Comment:
This is slightly unclear, since both `5` and `+5` are *technically* positive. The examples do make it clear however.

Reply:
Added a clearer explaination